WalletWasabi.Daemon/README.md

```diff
@@ -49,3 +50,9 @@ Run Wasabi Daemon and connect to the testnet Bitcoin network.
 ```bash
 $ WASABI-NETWORK=testnet wasabi.daemon
 ```
+
+Run Wasabi and open two wallets: AliceWallet and BobWallet
+
+```bash
+$ wasabi.daemon --wallet:AliceWallet,BobWallet
```

Comment:
There's one more way to break it - add a comma to the name.

Reply:
I've changed this to support multiple `--wallet=` arguments instead of only one with a separator. About the spaces and other characters, yes, we can wrap the whole argument around quotation marks or simply escape them individually.